### PR TITLE
Add browsable API templates for Helsinki Linked Courses

### PR DESCRIPTION
**Pull request by tuomas777**
Screenshot 2019-09-05 at 22 18 41

### CODE REVIEW

File: linkedevents/settings.py [hunk 1]
```diff
@@ -144,10 +147,11 @@ def get_git_revision_hash() -> str:
     'helusers.providers.helsinki',
 
     'helevents',
+    'helcourses',
     'munigeo',
     'leaflet',
     'django_orghierarchy',
-]
+] + env('EXTRA_INSTALLED_APPS')
```
Comment (Rikuoja) [inline linkedevents/settings.py]: Where do we need extra installed apps, I wonder?

Comment (vikoivun) [reply linkedevents/settings.py]: Däts for extensions?

Comment (Rikuoja) [reply linkedevents/settings.py]: Nope, there is `AUTO_ENABLED_EXTENSIONS = env('AUTO_ENABLED_EXTENSIONS')` for that?

Comment (vikoivun) [reply linkedevents/settings.py]: That enables extensions for every request, yes? Instead of activating by url parameter, yes?

Comment (Rikuoja) [reply linkedevents/settings.py]: Or, if this setting is meant to be changed whenever an extension is auto enabled, then surely `helcourses` should not be in the regular setup, but in the `env('EXTRA_INSTALLED_APPS')`?

Comment (vikoivun) [reply linkedevents/settings.py]: Ah. Indeed.

Comment (Rikuoja) [reply linkedevents/settings.py]: Yes it does. So, the question is, why do we need extra installed apps, if extensions are already installed?

Comment (tuomas777) [reply linkedevents/settings.py]: Linked Courses needs both `"extension_course"` in `INSTALLED_APPS` and `"course"` in `AUTO_ENABLED_EXTENSIONS`, the latter does nothing without the former. `AUTO_ENABLED_EXTENSIONS` is about auto enabling an installed extension on every request, just like Ville said. 

You are right that there is no need for `helcourses` to be installed by default, I'll change that.

Also, I'm not sure if this `EXTRA_INSTALLED_APPS` method is at all the best for allowing `INSTALLED_APPS` to be modified from env, feel free to propose a better solution :)

Comment (Rikuoja) [reply linkedevents/settings.py]: Okay, let's include instructions on how to install the courses extension as an example under `https://github.com/City-of-Helsinki/linkedevents#event-extensions`.

As courses is the only extension at the moment, we should just have instructions for that in the readme. The generic extension instructions do not really help anyone deploying courses.

**Comment by tuomas777** [comment]
Replaced by https://github.com/City-of-Helsinki/linkedevents/pull/321

**Pull request closed without merging**